howto/playAudio.md

```diff
@@ -1,16 +1,25 @@
-{% method %}
+# How To Play Audio On A Call {#top}
+
+## About {#about}
 
-# How to play audio on a call
+Play a .mp3 or .wav file on an active phone call. For more information about playing an audio file onto a call, visit the [full documentation](http://dev.bandwidth.com/ap-docs/methods/calls/postCallsCallIdAudio.html).
 
-### Before playing audio file
+## Assumptions
+* You have signed up for the [Bandwidth voice & messaging APIs](https://app.bandwidth.com)
+* You are familiar with [receiving incoming calls](incomingCallandMessaging.md) and [making outbound calls](outboundCall.md)
+
+## Steps
+**Steps 1 and 2 should be completed before using this tutorial**
 1. [Create call with callback URL](./outboundCall.md) -or- [Recieve Incoming Call](./incomingCallandMessaging.md)
-2. Make sure call is answered and active
+2. [Make sure call is answered and active]()
+3. [Play audio on call](#play-audio)
 
-### Play audio options
-The fileURL is the location of the audio file. Bandwidth currently supports WAV and MP3 files. [Click here](http://dev.bandwidth.com/ap-docs/methods/calls/postCallsCallIdAudio.html)for more information about play audio options.
+## Play audio on call {#play-audio}
 
-### Play audio file to the resource (ie. ...`/{userId}/calls/{callId}/`)
+#### Play audio options
+The fileURL is the location of the audio file. Bandwidth currently supports WAV and MP3 files. [Click here](http://dev.bandwidth.com/ap-docs/methods/calls/postCallsCallIdAudio.html)for more information about play audio options.
```

Comment:
General feedback, try to default to linking the relative words to other locations.  Like:

```markdown
Reference the [full documentation on playing audio in calls](http://dev.bandwidth.com/ap-docs/methods/calls/postCallsCallIdAudio.html) to learn more specifics.
```

It's not inherently bad, and sometimes works just fine.  ⚠️  Here is an overly opinionated article with an overall negative tone, but otherwise, good.   [Click here](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/)

Reply:
I used that one because it is in ap docs so can I use that here? I thought we essentially kept them separate